apps/etterlatte-beregning/src/main/kotlin/avkorting/Avkorting.kt

```diff
@@ -245,27 +266,16 @@ data class Avkorting(
         return avkortetYtelseMedAllForventetInntekt
     }
 
-	/*
-	 * Det er tilfeller hvor det er nødvendig å vite når første periode i inneværende begynner.
-	 * Ved inngangsår så vil ikke første måned nødvendgivis være januar så det må baseres på fom første periode.
-	 */
-    private fun foersteMaanedDetteAar() =
-        hentAarsoppgjoer()
-            .ytelseFoerAvkorting
-            .first()
-            .periode.fom
-
-    private fun hentAarsoppgjoer(): Aarsoppgjoer {
-        if (aarsoppgjoer.isEmpty()) {
-            return Aarsoppgjoer()
-        }
-        // TODO skal på sikt utlede etterspurt årsoppgjør
-        return aarsoppgjoer.single()
+    fun hentAarsoppgjoer(fom: YearMonth): Aarsoppgjoer {
+        val funnet = aarsoppgjoer.find { it.aar!! == fom.year }
```

Comment:
Hvor må det være assert non null (`!!`) for aar her?

Reply:
Henger igjen fra en tidligere commit hvor jeg gjorde feltet nullable med default null for å slippe å fikse kompilering av alle tester osv midlertidig.